src/guilds/GuildShowView.tsx

```diff
@@ -5,137 +5,61 @@ import {
-            Add Growth Exercise
-          </Button>
-        </Flex>
-        <Box mt="8px">
-          {showGrowthExercises && (
-            <HStack spacing={8}>
-              <Stack
-                spacing={3}
-                p={6}
-                bg="white"
-                borderWidth="1px"
-                borderRadius="lg"
-                alignItems="center"
-                boxShadow="sm"
-                width="260px"
-                height="200px"
-                cursor={"pointer"}
-                _hover={{ boxShadow: "md" }}
-                onClick={() => handleCreateGE("study")}
-              >
-                <Flex align={"center"}>
-                  <StudyExerciseIcon w={12} h={12} />
-                  <Text ml="8px" fontSize="lg">
-                    Study Exercise
-                  </Text>
-                </Flex>
-                <Text>Learn by researching and doing</Text>
-                <Text fontSize="sm" color="#006400">
-                  Avg Time: 5 hrs
-                </Text>
-              </Stack>
-              <Stack
-                spacing={3}
-                p={6}
-                bg="white"
-                borderWidth="1px"
-                borderRadius="lg"
-                alignItems="center"
-                boxShadow="sm"
-                width="260px"
-                height="200px"
-                cursor={"pointer"}
-                _hover={{ boxShadow: "md" }}
-                onClick={() => handleCreateGE("blog-article")}
-              >
-                <Flex align={"center"}>
-                  <BlogArticleIcon w={12} h={12} />
-                  <Text ml="8px" fontSize="lg">
-                    Blog Article
-                  </Text>
-                </Flex>
-                <Text>Solidify your understanding by writing</Text>
-                <Text fontSize="sm" color="#006400">
-                  Avg Time: 1hr
-                </Text>
-              </Stack>
-              <Stack
-                spacing={3}
-                p={6}
-                bg="white"
-                borderWidth="1px"
-                borderRadius="lg"
-                alignItems="center"
-                boxShadow="sm"
-                width="260px"
-                height="200px"
-                cursor={"pointer"}
-                _hover={{ boxShadow: "md" }}
-              >
-                <Flex align={"center"}>
-                  <TodayILearnedIcon w={12} h={12} />
-                  <Text ml="8px" fontSize="lg">
-                    Today I Learned
-                  </Text>
-                </Flex>
-                <Text>Log your day to day learnings</Text>
-                <Text fontSize="sm" color="#006400">
-                  Avg Time: 15 mins
-                </Text>
-              </Stack>
-            </HStack>
-          )}
-        </Box>
+
+      <GridItem my={"20px"} mx={"5%"} ml={"7%"}>
+        <MenuGrowthExercise handleCreateGE={handleCreateGE} title={title} guild={guild}/>
```

Comment:
So, since it is not a view, we dont need to have View in the name however I feel we can name this component better